### DISCUSSION TITLE
Welcome to avante.nvim Discussions!

### DISCUSSION THREAD

**Discussion by yetone** [yetone/avante.nvim#368] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by skw1335** [comment]
is there a free way to use this? lol

**Reply by aarnphm** [reply]
you can run your local model with ollama, or any openai compatible endpoint. Check wiki for more information

**Reply by thechillday1** [reply]
Get

**Reply by RubenSmn** [reply]
I am using the free version of Gemini Flash. See https://ai.google.dev/pricing

**Reply by danielju91** [reply]
Hello! Loving the outlook of this project. Hoping it will catch up to and surpass Cursor AI in the long run. @aarnphm A quick question about running a local model. Is it compatible with LM Studio?

**Reply by cliffcheney** [reply]
I am trying to get mlx_lm.server to work as an openai endpoint for avante. I want the Apple Silicon speed. Anyone heard of someone  getting this working?

**Comment by thechillday1** [comment]
> is there a free way to use this? lol

**Comment by mikayil-dev** [comment]
@yetone Amazing project! How long did it take you to build this?

**Reply by yetone** [reply]
It took a total of two days, one of which was spent on prompt engineering-related tasks.

**Reply by thechillday1** [reply]
Can you share your project with me?

**Comment by gurius** [comment]
Is it possible to configure the plugin to work with other local LLMs providers like LM Studio? I've tried configuring it, but my attempts were unsuccessful.

**Reply by aarnphm** [reply]
afaict it supports openai-compatible endpoint, so it should support.

**Comment by hyprsh** [comment]
Just wanna ask if there is a offical Discord?

 A lot of AI Tooling use Discord to stay in touch. Would be nice if avante also supports this way to talk about releases/ use cases and tips and tricks.

**Reply by aarnphm** [reply]
Hi there, running a discord is very hard to maintain. I refrain from doing this for now.

maybe once we have more ppl then sure, but github issues and PR will keep us focused.

**Reply by eyalk11** [reply]
I think that moderators are relatively easy to recurit. Though I don't see downside to having it , even if not moderated (unless in really exterme case) .

**Comment by thechillday1** [comment]
Fixed!

**Reply by peter-miroshnikov** [reply]
I am sorry, i deleted my comment here as i figured out is the wrong place to ask a question.
Could you point me to the place where the fix is done so i can get a better perspective and not ask stupid questions.

**Reply by thechillday1** [reply]
Alright

**Comment by MachineLearning-Nerd** [comment]
How can I limit input token from given file?
